### PR TITLE
Document CSSFontFeatureValuesRule

### PR DESCRIPTION
**Pull request by teoli2003**
### Description

Document `CSSFontFeatureValuesRule`.
Document `CSSFontFeatureValuesRule.fontFamily`

### Motivation

openwebdocs/project#152

### Additional details

This interface has additional properties. They are experimental and a recent addition to Chrome only (not even Edge and Opera). I didn't want to fall into this rabbit hole, as the current project goal is to document _interoperable_ features. 

### Related issues and pull requests

Some `mdn_url` entries are missing in BCD. I will create a PR over there soon.

### CODE REVIEW

File: files/en-us/web/api/cssfontfeaturevaluesrule/fontfamily/index.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+title: CSSFontFeatureValuesRule.fontFamily
+slug: Web/API/CSSFontFeatureValuesRule/fontFamily
+page-type: web-api-instance-property
+browser-compat: api.CSSFontFeatureValuesRule.fontFamily
+---
+
+{{ APIRef("CSSOM") }}
+
+The **`fontFamily`** property of the {{domxref("CSSConditionRule")}} interface represent the name of the font family it applies to.
```
Comment (dawei-wang) [inline files/en-us/web/api/cssfontfeaturevaluesrule/fontfamily/index.md:10]:
```suggestion
The **`fontFamily`** property of the {{domxref("CSSConditionRule")}} interface represents the name of the font family it applies to.
```

File: files/en-us/web/api/cssfontfeaturevaluesrule/fontfamily/index.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+## Value
+
+A string.
+
+## Examples
+
+### Read font family
+
+In this example, we declared to {{cssxref("@font-feature-values")}} one fo the _Font One_ font family, and the other for _Font Two_. We then uses the CSSOm to read these font families, displaying them into the log.
```
Comment (dawei-wang) [inline files/en-us/web/api/cssfontfeaturevaluesrule/fontfamily/index.md:20]:
```suggestion
In this example, we declared two {{cssxref("@font-feature-values")}} one for the _Font One_ font family, and the other for _Font Two_. We then use the CSSOM to read these font families, displaying them into the log.
```

File: files/en-us/web/api/cssfontfeaturevaluesrule/index.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+## Instance methods
+
+_Inherits methods from its ancestor {{domxref("CSSRule")}}._
+
+## Examples
+
+### Read font family
+
+In this example, we declared to {{cssxref("@font-feature-values")}} one fo the _Font One_ font family, and the other for _Font Two_. We then uses the CSSOm to read these font families, displaying them into the log.
```
Comment (dawei-wang) [inline files/en-us/web/api/cssfontfeaturevaluesrule/index.md:29]:
```suggestion
In this example, we declared two {{cssxref("@font-feature-values")}} one for the _Font One_ font family, and the other for _Font Two_. We then use the CSSOM to read these font families, displaying them into the log.
```

File: files/en-us/web/api/cssfontfeaturevaluesrule/index.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+The **`CSSFontFeatureValuesRule`** interface represents an {{cssxref("@font-feature-values")}} [at-rule](/en-US/docs/Web/CSS/At-rule), letting developpers assign for each font face a common name to specify features indices to be used in {{cssxref("font-variant-alternates")}}.
+
+{{InheritanceDiagram}}
+
+## Instance properties
+
+_Inherits properties from its ancestor {{domxref("CSSRule")}}._
+
+- {{domxref("CSSFontFeatureValuesRule.fontFamily")}}
+  - : A string identify the font-family this rule applies to.
```
Comment (dawei-wang) [inline files/en-us/web/api/cssfontfeaturevaluesrule/index.md:19]:
```suggestion
  - : A string that identifies the font-family this rule applies to.
```

**Review by dawei-wang — COMMENTED** [review]
A few typos

File: files/en-us/web/api/cssfontfeaturevaluesrule/fontfamily/index.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+## Value
+
+A string.
+
+## Examples
+
+### Read font family
+
+In this example, we declared two {{cssxref("@font-feature-values")}} one for the _Font One_ font family, and the other for _Font Two_. We then use the CSSOM to read these font families, displaying them into the log.
```
Comment (sideshowbarker) [inline files/en-us/web/api/cssfontfeaturevaluesrule/fontfamily/index.md:20]:
```suggestion
In this example, we declare two {{cssxref("@font-feature-values")}} one for the _Font One_ font family, and the other for _Font Two_. We then use the CSSOM to read these font families, displaying them into the log.
```

File: files/en-us/web/api/cssfontfeaturevaluesrule/index.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
+---
+title: CSSFontFeatureValuesRule
+slug: Web/API/CSSFontFeatureValuesRule
+page-type: web-api-interface
+browser-compat: api.CSSFontFeatureValuesRule
+---
+
+{{APIRef("CSSOM")}}
+
+The **`CSSFontFeatureValuesRule`** interface represents an {{cssxref("@font-feature-values")}} [at-rule](/en-US/docs/Web/CSS/At-rule), letting developpers assign for each font face a common name to specify features indices to be used in {{cssxref("font-variant-alternates")}}.
```
Comment (sideshowbarker) [inline files/en-us/web/api/cssfontfeaturevaluesrule/index.md:10]:
```suggestion
The **`CSSFontFeatureValuesRule`** interface represents an {{cssxref("@font-feature-values")}} [at-rule](/en-US/docs/Web/CSS/At-rule), letting developers assign for each font face a common name to specify features indices to be used in {{cssxref("font-variant-alternates")}}.
```

File: files/en-us/web/api/cssfontfeaturevaluesrule/index.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+
+{{InheritanceDiagram}}
+
+## Instance properties
+
+_Inherits properties from its ancestor {{domxref("CSSRule")}}._
+
+- {{domxref("CSSFontFeatureValuesRule.fontFamily")}}
+  - : A string that identifies the font-family this rule applies to.
```
Comment (sideshowbarker) [inline files/en-us/web/api/cssfontfeaturevaluesrule/index.md:19]:
```suggestion
  - : A string that identifies the font family this rule applies to.
```

File: files/en-us/web/api/cssfontfeaturevaluesrule/index.md [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+
+## Instance methods
+
+_Inherits methods from its ancestor {{domxref("CSSRule")}}._
+
+## Examples
+
+### Read font family
+
+In this example, we declared two {{cssxref("@font-feature-values")}} one for the _Font One_ font family, and the other for _Font Two_. We then use the CSSOM to read these font families, displaying them into the log.
```
Comment (sideshowbarker) [inline files/en-us/web/api/cssfontfeaturevaluesrule/index.md:29]:
```suggestion
In this example, we declare two {{cssxref("@font-feature-values")}} one for the _Font One_ font family, and the other for _Font Two_. We then use the CSSOM to read these font families, displaying them into the log.
```

**Review by sideshowbarker — APPROVED** [review]

**Pull request merged**